ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -9,8 +9,8 @@
       OPENSTACK_IMG: {{ operators_build_output[cifmw_operator_build_meta_name].image_catalog }}
       {% endif %}
     cifmw_edpm_prepare_make_openstack_deploy_env: |
+      OPENSTACK_BRANCH: "main"
```

Comment:
Under wich circunstances are you hitting errors here. The main purpose of a "" here is to point the branch and HEAD to the one the git_src_dir points. This is usually used only if building an operator from a PR.

Reply:
cifmw_installyamls_repos: "{{ ansible_user_dir }}/install_yamls" might not be followed by some parts. I'll make cleaner setup.

~/src/github/openstack-k8s-operators/install_yamls was used from old times.

Reply:
Not needed at the end.